libraries/session-storage/impl/src/main/sqldelight/migrations/0.sqm

```diff
@@ -1,3 +1,6 @@
+-- This file is not striclty necessary, since the first
```

Comment:
Perhaps this file should be removed.

Reply:
I wanted to remove it, but then I got a warning in the IDE in the file 1.sqm, claiming that `SessionData does not exist`.

So I decided to keep it and add a comment. It's fine to have it anyway I think.

Reply:
> I wanted to remove it, but then I got a warning in the IDE in the file 1.sqm, claiming that `SessionData does not exist`.
> 
> So I decided to keep it and add a comment. It's fine to have it anyway I think.

This might be a "bug" due to the fact that our Sqldelight version is lower than the current one.
So you're ide might be running the sqldelight plugin 2.0.0 but our project is on 1.5.5

We might consider upgrading to 2.0.0  it's been released months ago.

Reply:
Migration is not that difficult, just a few package renames.

Reply:
https://github.com/vector-im/element-x-android/issues/1557